From d2e21fb6fbd7900cb02540ec9fa9e48f91079ce1 Mon Sep 17 00:00:00 2001
From: Filip Pytloun <filip@pytloun.cz>
Date: Fri, 29 Jan 2016 16:24:42 +0100
Subject: Do not require setuptools-scm

---
 setup.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/setup.py b/setup.py
index 18e6e8e..713d786 100644
--- a/setup.py
+++ b/setup.py
@@ -54,10 +54,6 @@ class PrintRequirements(Command):
 
 setup(
     name='vdirsyncer',
-    use_scm_version={
-        'write_to': 'vdirsyncer/version.py',
-    },
-    setup_requires=['setuptools_scm'],
     author='Markus Unterwaditzer',
     author_email='markus@unterwaditzer.net',
     url='https://github.com/pimutils/vdirsyncer',
